pinot-core/src/main/java/org/apache/pinot/core/transport/server/routing/stats/ServerRoutingStatsManager.java

```diff
@@ -198,6 +198,10 @@ private void updateStatsUponResponseArrival(String serverInstanceId, long latenc
     }
   }
 
+  public Map<String, ServerRoutingStatsEntry> getServerRoutingStats() {
+    return _serverQueryStatsMap;
```

Comment:
Let us check for `_isEnabled` here? _serverQueryStatsMap would be null if config is disabled.

Reply:
Done

Reply:
Oh wait - I checked for `isEnabled` in the `PinotBrokerDebug` file. Is that OK ? Right now, it is the caller's responsibility to check for `isEnabled` and null.